core/scripts/ocr2vrf/main.go

```diff
@@ -216,16 +230,16 @@ func main() {
 
 	case "coordinator-redeem-randomness":
 		cmd := flag.NewFlagSet("coordinator-redeem-randomness", flag.ExitOnError)
-		coordinatorAddress := cmd.String("coordinator-address", "", "VRF beacon coordinator contract address")
+		coordinatorAddress := cmd.String("coordinator-address", "", "VRF coordinator contract address")
 		requestID := cmd.Int64("request-id", 0, "request ID")
 		helpers.ParseArgs(cmd, os.Args[2:], "coordinator-address", "request-id")
 		redeemRandomness(e, *coordinatorAddress, big.NewInt(*requestID))
 
-	case "coordinator-info":
+	case "beacon-info":
 		cmd := flag.NewFlagSet("coordinator-info", flag.ExitOnError)
-		coordinatorAddress := cmd.String("coordinator-address", "", "VRF beacon coordinator contract address")
+		coordinatorAddress := cmd.String("coordinator-address", "", "VRF coordinator contract address")
 		helpers.ParseArgs(cmd, os.Args[2:], "coordinator-address")
-		coordinator := newVRFBeaconCoordinator(common.HexToAddress(*coordinatorAddress), e.Ec)
+		coordinator := newVRFBeacon(common.HexToAddress(*coordinatorAddress), e.Ec)
 		keyID, err := coordinator.SKeyID(nil)
 		helpers.PanicErr(err)
 		fmt.Println("coordinator key id:", hexutil.Encode(keyID[:]))
```

Comment:
Should we find/replace coordinator with `beacon` here, e.g `coordinatorAddress` -> `beaconAddress`, `coordinator` -> `beacon`, etc.

Reply:
yeah nice catch. updated.